blog/2024-01-23/index.md

```diff
@@ -0,0 +1,58 @@
+
+This methodology reveals that over the past few months there have over a million updates of each of the latest FreeDesktop SDK runtime releases, meaning we've passed the one million active user mark.
+
+:::note
+If anything, we believe this to be a conservative estimate as some users may be using apps that do not use the specific runtime measured.
+:::
+
+If you'd like to explore more of this data yourself, there's this [handy dashboard by Kristian Klausen](https://klausenbusk.github.io/flathub-stats/#ref=org.freedesktop.Platform/23.08&interval=infinity&downloadType=updates) ([source](https://github.com/klausenbusk/flathub-stats)), or you can always chat with us in [#flathub on Matrix](https://matrix.to/#/#flathub:matrix.org).
+
+## What's Driving Growth
+
+We attribute the growth of Flathub to a few developments over the years, and especially the last several months:
+
+1. **Popular app availability**: Since we started Flathub, we've seen apps like Firefox, Google Chrome, Discord, VLC, Spotify, Telegram, Microsoft Edge, Steam, OBS Studio, Zoom, Thunderbird, and _many_ more make their way into the store. These are many of the apps people want and expect, especially if they're coming to a Linux desktop for the first time. Each time another popular app hits Flathub, it makes Flathub that much more compelling to users.
+
+2. **Verified apps**: We've heard that some folks have held off installing specific apps or even using Flathub altogether because they didn't want a third-party maintainer redistributing an app they rely on—verifying apps solves this by assuring users their favorite app is actually coming from its developer. With an ever-increasing number of apps choosing to get verified—including both big names like Firefox, Discord, and Telegram as well as many newer indie apps—the trust and adoption of Flathub increases.
+
+3. **Steam Deck**: It's not [just a meme](https://mastodon.blaede.family/@cassidy/111031129234702967); we suspect Flathub being included as the default app source for the Steam Deck's desktop mode has had a large positive effect on the usage of Flathub. Just look at some of the [most popular apps](https://flathub.org/apps/collection/popular): retro game emulators, game compatibility tools, gaming-oriented chat services, alternate game launchers—all the kinds of apps you'd expect Steam Deck users to want. And according to our category [statistics](https://flathub.org/statistics), "games" itself is the most downloaded category of app on Flathub. It turns out selling ["multiple millions" of devices](https://www.theverge.com/2023/11/9/23954205/valve-steam-deck-multiple-millions) has an impact on the ecosystem!
```

Comment:
> And according to our category [statistics](https://flathub.org/statistics), "games" itself is the most downloaded category of app on Flathub.

It might be confusing, but that are not download counts. That's apps per category. So how many apps are in each category (technically an app can declare multiple but you can probably ignore that)

Reply:
ooooooh yeah I totally missed that. I can reword that a bit:

```suggestion
3. **Steam Deck**: It's not [just a meme](https://mastodon.blaede.family/@cassidy/111031129234702967); we suspect Flathub being included as the default app source for the Steam Deck's desktop mode has had a large positive effect on the usage of Flathub. Just look at some of the [most popular apps](https://flathub.org/apps/collection/popular): retro game emulators, game compatibility tools, gaming-oriented chat services, alternate game launchers—all the kinds of apps you'd expect Steam Deck users to want. And in fact, "game" is the single most common category of app on Flathub. It turns out selling ["multiple millions" of devices](https://www.theverge.com/2023/11/9/23954205/valve-steam-deck-multiple-millions) has an impact on the ecosystem!
```